implementations/rust/ockam/ockam_command/src/util/mod.rs

```diff
@@ -398,6 +398,31 @@ where
     Ok(r)
 }
 
+pub fn embedded_node_daemon<A, F, Fut, T>(f: F, a: A) -> crate::Result<T>
```

Comment:
@adrianbenavides `daemon` leads me to to think of an OS level process here. Since we're not doing that here let's call this function something else.

Reply:
I don't really know what to call it. The idea is that this function creates an embedded node and the context that is used is not stopped, so it will run indefinitely. I don't know the word for that :laughing:

Reply:
I'm a fan of very explicit names `embedded_node_that_is_not_stopped`

Reply:
I like that!